### PR TITLE
fix: comment back non-required-shapefiles and disable upload of non-z…

### PR DESCRIPTION
**Pull request by Thuhaa**
…ipped gdb

Thank you for submitting a pull request!

## Description
I commented back the non-necessary shapefile extensions except `.atx` to disable upload of non zipped ESRI geodatabases

### Type of Pull Request

* [ ] Adding a feature
* [x] Fixing a bug
* [ ] Maintaining documents
* [ ] Adding tests
* [ ] Others ()

### Verify the followings

* [x] Code is up-to-date with the `develop` branch
* [x] No build errors after `pnpm build`
* [x] No lint errors after `pnpm lint`
* [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`
* [x] Make sure all the existing features working well

### Changesets

* [ ] If your PR makes a change under `packages` folder that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

Refer to [CONTRIBUTING.MD](https://github.com/UNDP-Data/geohub/blob/develop/CONTRIBUTING.md) for more information.

┆Issue is synchronized with this [Wrike task](https://www.wrike.com/open.htm?id=1259869971) by [Unito](https://www.unito.io)

### CODE REVIEW

File: sites/geohub/src/lib/config/AppConfig/AcceptedExtensions.ts [hunk 1]
```diff
@@ -37,17 +37,16 @@ export const AccepedExtensions: AcceptedExtension[] = [
...
-			// 'cpg',
-			// 'sbn',
-			// 'sbx',
-			// 'fbn',
-			// 'fbx',
-			// 'ain',
-			// 'aih',
-			// 'ixs',
-			// 'mxs'
-			// 'atx'
```
Comment (JinIgarashi) [inline sites/geohub/src/lib/config/AppConfig/AcceptedExtensions.ts:50]: You shoudnot remove atx extension

**Review by JinIgarashi — CHANGES_REQUESTED** [review]

**Comment by JinIgarashi** [comment]
You can check folder name of .atx file and skip adding if folder name contains .gdb name

File: sites/geohub/src/routes/(app)/data/upload/+page.svelte [hunk 2]
```diff
@@ -119,6 +131,19 @@
...
 							})
 						);
+					} else {
+						zipEntryPromises.push(
+							zipEntry.async('blob').then((blob) => {
+								zipFileList.push({
+									name: zipEntry.name,
+									path: zipFile.path
+										? zipEntry.name
+										: `${zipFile.name.replace('.zip', '')}/${zipEntry.name}`,
```
Comment (JinIgarashi) [inline sites/geohub/src/routes/(app)/data/upload/+page.svelte:141]: probably you need to check other archived formats as well (.7z, .tar, etc)?

File: sites/geohub/src/routes/(app)/data/upload/+page.svelte [hunk 3]
```diff
@@ -328,10 +365,46 @@
...
+			if (file.path) {
+				const filePath = file.path;
+				/**
+				 * If the file is a file inside a geodatabase, do not return it
+				 * This is because the geodatabase should be uploaded as a zip file and never as single files
+				 * Reject .atx files that don't have corresponding file names
+				 */
+				if (
+					filePath.split('/').length > 1 &&
+					filePath.includes('.gdb') &&
```
Comment (JinIgarashi) [inline sites/geohub/src/routes/(app)/data/upload/+page.svelte:382]: for `.mdb`? is `.mdb` also a folder?

**Review by JinIgarashi — CHANGES_REQUESTED** [review]
source code looks much better. I just commented two things.

File: sites/geohub/src/routes/(app)/data/upload/+page.svelte (hunk 2 above)
Comment (Thuhaa) [reply sites/geohub/src/routes/(app)/data/upload/+page.svelte:141]: @JinIgarashi 
1. If the user selects single file shapefiles example `sdr_subnational.shp`, `sdr_subnational.prj` they are immediately zipped using `JsZip` into a file named `sdr_subnational.zip` which is displayed on the table. Any corresponding shapefile files that are selected are added to this zipfile
2. The highlighted code is used inside the `getZipFilesList` which is in turn used in the `checkShapeFileValidity` function which returns a mapping in the format `{"shapefile_name":["list","of","missing","shapefiles"]}`
3. The `checkShapefile` validity checks for shapefile validity in these zip files, _**not the ones added as zip files (or any other format) directly by the user.**_. 
4. We don't zip the shapefiles into any other archive format. **_Only_** `.zip` as currently there are no more checks for archive files uploaded by the user, but rather only the shapefiles that have been zipped by GeoHub. Only shapefiles that have been uploaded by the user as single files are zipped

**Pull request merged**